### ISSUE TITLE
WordCount plugin crashes on ngOnDestroy

### ISSUE THREAD

**Issue by Trysor** [ckeditor/ckeditor5-angular#123]
Angular project package.json:
```json
"zone.js": "git+https://git@github.com/angular/zone.js.git",
"@ckeditor/ckeditor5-angular": "1.1.0",
[...]
```

CKEditor 5 custom build package.json:
```json
	"dependencies": {
		"@ckeditor/ckeditor5-alignment": "11.1.2",
		"@ckeditor/ckeditor5-basic-styles": "11.1.2",
		"@ckeditor/ckeditor5-block-quote": "11.1.1",
		"@ckeditor/ckeditor5-editor-classic": "12.1.2",
		"@ckeditor/ckeditor5-essentials": "11.0.3",
		"@ckeditor/ckeditor5-font": "11.2.0",
		"@ckeditor/ckeditor5-heading": "11.0.3",
		"@ckeditor/ckeditor5-image": "13.1.1",
		"@ckeditor/ckeditor5-link": "11.1.0",
		"@ckeditor/ckeditor5-list": "12.0.3",
		"@ckeditor/ckeditor5-media-embed": "11.1.2",
		"@ckeditor/ckeditor5-paragraph": "11.0.3",
		"@ckeditor/ckeditor5-remove-format": "10.0.2",
		"@ckeditor/ckeditor5-table": "13.0.1",
		**"@ckeditor/ckeditor5-word-count": "10.0.0",**
		"@ckeditor/ckeditor5-theme-lark": "14.1.0"
	},
	"devDependencies": {
		"@ckeditor/ckeditor5-dev-utils": "12.0.1",
		"@ckeditor/ckeditor5-dev-webpack-plugin": "8.0.1",
                [....]
       }
}
```
CKEditor 5 custom build (updated as of v12.3.0), with WordCount included. Upon navigating away from the angular route containing CKEditor, the `ckeditor5-angular` component is destroyed giving the following error:

```
ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'element' of undefined
TypeError: Cannot read property 'element' of undefined
at db.builtinPlugins.destroy
```

Where destroy contains:
```
destroy() {
    **this._outputView.element.remove(),**
    this._outputView.destroy(),
    super.destroy()
}
```
The bolded line is where DevTools stopped.

In this example the outputview element is not used and was never injected into DOM. I am instead using the event programmatically.

**Comment by ma2ciek** [comment]
Hi!

I wasn't able to reproduce the issue. Could you provide a simple app where I could examine the error?

**Comment by Trysor** [comment]
repro: https://github.com/Trysor/ckeditorangular

**Comment by ma2ciek** [comment]
The error is caused by the fact that you don't have the word-count plugin properly configured. See:
- https://ckeditor.com/docs/ckeditor5/latest/features/word-count.html
- https://github.com/ckeditor/ckeditor5-word-count/issues/16

**Comment by ma2ciek** [comment]
Actually, the bug is partially caused by us. See the second link for more information.

**Comment by Trysor** [comment]
The word count plugin should be usable without having to instantiate the word count container, in my eyes. Thanks for the followup!

**Comment by ma2ciek** [comment]
> The word count plugin should be usable without having to instantiate the word count container, in my eyes.

Yep, definitely!

> Thanks for the followup!

No problem, the bug should be fixed soon, but the package will be released after ~1-1.5 months, unfortunately.

**Comment by dyh333** [comment]
@ma2ciek  I am trying to use wordCount plugin in angular, is anywhere provide a simple demo?
